sc2reader/events/game.py

```diff
@@ -38,7 +38,10 @@ def __init__(self, frame, pid):
 
     def _str_prefix(self):
         if self.player:
-            player_name = self.player.name if getattr(self, 'pid', 16) != 16 else "Global"
+            if not self.player.name:
```

Comment:
i think this needs to be flipped since if `getattr(self, 'pid', 16) == 16 and not self.player.name`, then it will do `Player 16 - (?)` instead of `Global` like it does now. 

it will probably be easiest if you break up the ternary statement into multiple checks since we now have multiple branches:

```
if pid == 16:
  ...
elif self.player.name:
   ...
else:
   ...
```

Reply:
I split it out into multiple branches and built a test using a Mock class. I thought about using unittest.mock, but it would require more python 2/3 specialized handling.